src/main/resources/Changelog.txt

```diff
@@ -1,6 +1,7 @@
 5.10:
   - Remove Towny dependency and blind convert town/nation accounts.
   - Refactor event system to support pre 1.14 servers.
+  - Add support for Towny debt accounts.
```

Comment:
It might be more accurate to add that this is for importing debt accounts from esseco, iconomy already supported the debt accounts themselves.

Reply:
done